<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_build</class>
 <widget class="QWidget" name="form_build">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="3" column="1" colspan="2">
          <widget class="QTextEdit" name="txt_build_arg"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="ln_build_dir"/>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="btn_build_dir">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="lbl_build_dir">
           <property name="text">
            <string>Working Directory</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lbl_build_args">
           <property name="text">
            <string>Arguments</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="ln_build_app"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lbl_build_program">
           <property name="text">
            <string>Program File</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btn_build_app">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="3">
          <widget class="QPushButton" name="btn_build">
           <property name="text">
            <string>Build</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="6">
          <widget class="QListWidget" name="list_build_db"/>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QListWidget" name="list_build"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_buildadd">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="insert-table-row">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_buildremove">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="removecell">
             <normaloff>.</normaloff>.</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QTreeView" name="tw_compile">
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lbl_compile_import_local">
         <property name="text">
          <string>Directory:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ln_compile_import_local"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_compile_import_local">
         <property name="text">
          <string>Import from local</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_compile_import_clone">
         <property name="text">
          <string>Git clone:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ln_compile_import_remote"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_compile_import_branch">
         <property name="text">
          <string>Branch:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ln_compile_import_branch">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_compile_import_remote">
         <property name="text">
          <string>Import from remote</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
